esm-samples/jsapi-custom-workers/package.json

```diff
@@ -12,7 +10,7 @@
     "html-webpack-plugin": "^5.5.0",
     "mini-css-extract-plugin": "^2.7.2",
     "npm-run-all": "^4.1.5",
-    "rollup": "^3.10.1",
+    "rollup": "^3.16.0",
```

Comment:
Update?

Reply:
Oh, I was doing before/after testing and accidentally checked in the wrong version. I'll update and retest for this sample and jsapi-node.